samples/drivers/soc_flash_nrf/src/main.c

```diff
@@ -67,16 +64,12 @@ int main(void)
 {
 	const struct device *flash_dev = TEST_PARTITION_DEVICE;
 	struct flash_parameters flash_params;
-	uint32_t buf_array_1[4] = { TEST_DATA_WORD_0, TEST_DATA_WORD_1,
-				    TEST_DATA_WORD_2, TEST_DATA_WORD_3 };
-	uint32_t buf_array_2[4] = { TEST_DATA_WORD_3, TEST_DATA_WORD_1,
-				    TEST_DATA_WORD_2, TEST_DATA_WORD_0 };
-	uint32_t buf_array_3[8] = { TEST_DATA_WORD_0, TEST_DATA_WORD_1,
-				    TEST_DATA_WORD_2, TEST_DATA_WORD_3,
-				    TEST_DATA_WORD_0, TEST_DATA_WORD_1,
-				    TEST_DATA_WORD_2, TEST_DATA_WORD_3 };
-	uint32_t buf_word = 0U;
+
+	const uint8_t test_data[16] = {0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08,
```

Comment:
Make this a little bigger so that it can handle multiple write-block-sizes, that we know for our soc for now.